docs/software/admin.md

```diff
@@ -801,13 +799,6 @@ Stellar-core can be gracefully exited at any time by delivering `SIGINT` or
    and you may need to remove the file before it will restart. 
    Otherwise, all components are designed to recover from abrupt termination.
 
-Stellar-core can also be packaged in a container system such as Docker, so long 
-as `BUCKET_DIR_PATH`, `TMP_DIR_PATH`, and the database are stored on persistent 
```

Comment:
you removed too much here: the requirement for persistent volumes still exist

Reply:
you forgot to address this comment (don't remove Docker & persistent volume mention)

Reply:
Sorry! Weird, I didn't see this comment originally.

Reply:
no problem it happens.
You missed the note following this paragraph too btw (look for TMP_DIR_PATH)